src/auth/index.ts

```diff
@@ -173,6 +179,8 @@ export class Auth {
         publicKey: keys.publicKey,
         privateKey: keys.privateKeyEncrypted,
         revocationKey: keys.revocationCertificate,
+        publicKyberKey: keys.publicKyberKey,
```

Comment:
To avoid the fields explosion, let's use a cleaner object organization adding a new field called `keys`: 
```
{
 keys: {
   [algorithName]: {
     privateKey: ..., 
     publicKey: ...,
   }
 }
}
```
Leave the `privateKey` and `publicKey` fields for compatibility but set a comment above mentioning it is deprecated.

Reply:
Done. Maybe we move the revocationKey key inside ecc too? The current structure is: 
```
revocationKey: ..., 
keys: {
   ecc: {
     privateKey: ..., 
     publicKey: ...,
   }, 
   kyber: {
     privateKey: ..., 
     publicKey: ...,
   }, 
 }
```